bigtable/snippets/writes/write_conditionally.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+# Copyright 2019, Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# [START bigtable_writes_conditional]
+import datetime
+
+from google.cloud import bigtable
+from google.cloud.bigtable.row_filters import ColumnQualifierRegexFilter, \
```

Comment:
Nit: Prefer importing modules not classes. i.e. `from google.cloud.bigtable import row_filters` and use `row_filters.ColumnQualifierRegexFilter` in the sample.